src/main/java/org/apache/xml/security/encryption/XMLCipherUtil.java

```diff
@@ -274,13 +274,18 @@ public static KeyAgreementParameters constructAgreementParameters(String agreeme
      * @param keyDerivationMethod element with the key derivation method data
      * @param keyBitLength        expected derived key length in bits
      * @return KeyDerivationParameters data
-     * @throws XMLSecurityException if the invalid key derivation parameters are provide
-     * @throws XMLEncryptionException if the invalid key derivation is not supported
+     * @throws XMLEncryptionException throwen in case if KDFParams cannot be created or the
```

Comment:
typo: "throwen", but I would just remove "thrown in case" as that is implied. Also, a nit but you don't need to end with a period.

Reply:
Thanks for the warning about the typo, I fixed the description now.